src/transformers/modeling_bart.py

```diff
@@ -287,9 +290,7 @@ def forward(
         # check attention mask and invert
         if attention_mask is not None:
             assert attention_mask.dim() == 2
-
-            attention_mask = (1.0 - attention_mask.long()) * -10000.0
-            assert attention_mask.max() <= 0
+            attention_mask = attention_mask.eq(0).to(MASK_DTYPE)
```

Comment:
isn't the output of `.eq()` already in the type of MASK_DTYPE? (bool or uint8 depending on the PyTorch version)

Reply:
Yes, great catch.